server/utils/rating.util.js

```diff
@@ -0,0 +1,54 @@
+import db from '../models';
+
+const { Article } = db;
+
+class Rating {
+  /**
+   * Get the claps for an article
+   * @param {string} id
+   * @returns {number} claps
+   */
+  static async getArticleClaps(id) {
+    return 30;
```

Comment:
Why did you return 30 in this function?

Reply:
It served as a dummy value. We will implement the method fully once we have the clap feature merged and we rebase